packages/swarm-components/src/Button.jsx

```diff
@@ -119,4 +119,59 @@ const Button = (props: ButtonProps): React.Element<'button'> => {
 	);
 };
 
+// duplicated with a forwardRef for use in Dropwdown internally
+// not exported from swarm-ui
+// open to suggestions moving forward that don't introduce breaking changes
```

Comment:
Ah, it won't work as is since button is functional and doesn't have an instance for `ref` to work on but if we convert it to a class component it should.

Reply:
ah i see. I think reducing duplication would be best - could you just rename the prop to `buttonRef` or something?

Reply:
yea, got it to work by calling it `forwardedRef`